lib/public/Mail/IMessage.php

```diff
@@ -93,4 +93,14 @@ public function setBcc(array $recipients): IMessage;
 	 * @since 13.0.0
 	 */
 	public function useTemplate(IEMailTemplate $emailTemplate): IMessage;
+
+	/**
+	 * Add the Auto-Submitted header to the email, preventing most automated
+	 * responses to automated messages.
+	 *
+	 * @param string $value (one of AutoSubmittedValue::NO, AutoSubmittedValue::AUTO_GENERATED, AutoSubmittedValue::AUTO_REPLIED)
```

Comment:
```suggestion
	 * @param AutoSubmittedValue::* $value (one of AutoSubmittedValue::NO, AutoSubmittedValue::AUTO_GENERATED, AutoSubmittedValue::AUTO_REPLIED)
```

https://psalm.dev/docs/annotating_code/typing_in_psalm/#specifying-stringint-options-aka-enums

Reply:
alternatively leave `string $value` and *add* `@psalm-param` in another line